1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,33 @@
+import { fizzBuzz } from './fizzbuzz'
 
 describe("fizzbuzz", () => {
 
+    it('returns a string', () => {
+        expect(typeof fizzBuzz(5)).toBe('string')
+    })
+
+    it('returns Fizz for 3', () => {
+        expect(fizzBuzz(3)).toEqual('Fizz')
+    })
+
+    it('returns Buzz for 5', () => {
+        expect(fizzBuzz(5)).toEqual('Buzz')
+    })
+
+    it('returns FizzBuzz for 3 and 5', () => {
+        expect(fizzBuzz(15)).toEqual('FizzBuzz')
+    })
+
+    it('takes numbers from 1 to 100', () => {
+        expect(fizzBuzz(5)).toBeTruthy()
+    })
+
+    it('does not accept numbers less than 1', () => {
+        expect(fizzBuzz(-1)).toBeFalsy()
+    })
+
+    it('does not accept numbers greater than 101', () => {
+        expect(fizzBuzz(101)).toBeFalsy()
+    })
```

Comment:
As a suggestion, you might be able to come up with a more generic name for these two, and use parameterization:

```ts
it.each([[-1], [101]])('more generic name where %p returns false', (numberArg) => {
  expect(fizzBuzz(numberArg)).toBeFalsy()
})
```